VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -90,19 +99,19 @@ final internal class ResponseCache
             self.cache.setObject(responseDictionary, forKey: key)
         }
         
-        private func responseDictionaryForKey(key: String) -> VimeoClient.ResponseDictionary?
+        private func responseDictionary(forKey key: String, completion: ResponseDictionaryClosure)
```

Comment:
Agreed, but this function is an implementation of a function in the `Cache` protocol. While this memory cache isn't async, the disk cache is. I'd rather return the response in a block in the memory cache and have the two caches follow a shared contract than have different getter functions for each.